fbw-common/src/systems/instruments/src/ND/shared/RadioNavInfo.tsx

```diff
@@ -231,6 +230,10 @@ class AdfInfo extends DisplayComponent<{ bus: EventBus, index: 1 | 2, visible: S
         this.adfFrequency,
     );
 
+    private readonly visibilitySub = MappedSubject.create(
```

Comment:
`isVisible` is a bit nicer and more descriptive than `visibilitySub`. Hungarian notation is bad too 😛 .